src/Main.hs

```diff
@@ -32,51 +35,51 @@ options = Options
       <*> optional ( argument str
           ( metavar "CONTRACT"
          <> help "Contract inside of file to analyze" ))
-      <*> optional ( strOption
-          ( long "solc-args"
-         <> help "Optional solidity compiler arguments" ))
-      <*> optional ( option auto
-          ( long "epochs"
-         <> help "Optional number of epochs to run coverage guidance" ))
+      <*> optional ( option str
```

Comment:
I'm not sure how I feel about file-based conf being the only option. Can we keep the CLI options?